### PR TITLE
Resolves #10

### LINKED ISSUE #10: Functional GUI with ability to set keybinds
- User will be able to set keybinds

### PR DESCRIPTION
**Pull request by pmdwight**
Added ModeForm which will be accessed when choosing a specific mode from main form (maybe named AllModesForm or something of the likes). Added architecture for setting keybinds for Hot Keys. See TODO's for my strategy on how to combine with others' work. ModKeys.cs is the same as enum ModifierKeys in yours, Sean- so will only need one. Also ModeForm was refactored from Form1 so we will need to handle the merging of that (create new Form to house your Form1 or my ModeForm).

### CODE REVIEW

File: HotKey/HotKey_MainFolder/ModeForm.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            InitializeHotKeyControls();
+
+            modeLabel.Text = modeName;
+        }
+
+        private void InitializeHotKeyControls()
+        {
+            foreach (HotKeyItem hotKeyItem in hotKeyItemList)
+            {
+                hotKeyItemPanel.Controls.Add(new HotKeyControl(hotKeyItemList, hotKeyItem));
```
Comment (cyrokamogawa) [inline HotKey/HotKey_MainFolder/ModeForm.cs]: What is hotKeyItemPanel, and what is this for loop doing?

**Review by cyrokamogawa — DISMISSED** [review]

File: HotKey/HotKey_MainFolder/ModKeys.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HotKey_MainFolder
+{
+    [Flags]
```
Comment (ghost) [inline HotKey/HotKey_MainFolder/ModKeys.cs:9]: Do we want to use this for the code I uploaded? Repetitive for us to create it twice. See my comment on storing the keys in one main class.

File: HotKey/HotKey_MainFolder/Program.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new ModeForm());
```
Comment (ghost) [inline HotKey/HotKey_MainFolder/Program.cs]: Maybe change from ModeForm to something more specific just for clarity. Took me a minute to figure out which this is.

File: HotKey/HotKey_MainFolder/HotKeyControl.resx [hunk 4]
```diff
@@ -0,0 +1,120 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ghost) [inline HotKey/HotKey_MainFolder/HotKeyControl.resx:1]: I like the choice of WPF definitely looks a lot cleaner than winforms. Nice idea.

**Review by ghost — DISMISSED** [review]
I left a few comments about possible changes, but this looks clean. Nice job bud.

**Comment by pmdwight** [comment]
Closing this pull request, will reopen new one with updates.

**Pull request closed without merging**